extensions/terminal-suggest/src/terminalSuggestMain.ts

```diff
@@ -248,6 +247,27 @@ export async function getCompletionItemsFromSpecs(
 					command.documentation
 				));
 				labels.add(commandTextLabel);
+			} else {
+				// Potential builitin that should override the spec's description
+				let score = typeof command.label === 'object' ? (command.label.detail !== undefined ? 2 : 0) : 0;
+				score += typeof command.label === 'object' ? (command.label.description !== undefined ? 1 : 0) : 0;
+				if (score > 0) {
+					const existingItem = items.find(i => typeof i.label === 'string' ? i.label : i.label.label === commandTextLabel);
+					if (existingItem) {
+						score -= typeof existingItem.label === 'object' ? (existingItem.label.detail !== undefined ? 2 : 0) : 0;
+						score -= typeof existingItem.label === 'object' ? (existingItem.label.description !== undefined ? 1 : 0) : 0;
+						if (score >= 0) {
+							items.splice(items.indexOf(existingItem), 1);
+							items.push(createCompletionItem(
```

Comment:
May as well insert via the splice call, when just swapping an item out V8 might have optimizations to avoid bigger changes in the data structure